src/protocols/kubernetes/settings.c

```diff
@@ -275,6 +281,11 @@ guac_kubernetes_settings* guac_kubernetes_parse_args(guac_user* user,
         guac_user_parse_args_string(user, GUAC_KUBERNETES_CLIENT_ARGS, argv,
                 IDX_CONTAINER, NULL);
 
+    /* Read exec command (optional) */
+    settings->exec_command =
+        guac_user_parse_args_string(user, GUAC_KUBERNETES_CLIENT_ARGS, argv,
+                IDX_EXEC_COMMAND, NULL);
```

Comment:
The string returned by `guac_user_parse_args_string()` is dynamically-allocated and will need to eventually be freed. See:

https://github.com/apache/guacamole-server/blob/2aa0218cc761b0810d20d92ba3deccc57c993a63/src/protocols/kubernetes/settings.c#L399-L429

Reply:
Done